### ISSUE TITLE
Print metric on screen per epoch instead of per step

### ISSUE THREAD

**Issue by jrzaurin** [tflearn/tflearn#881]
This might be (or surely is) a trivial question, but I don't seem to find the solution in the documentation or online. I simply want to change the logs to be printed on screen when using tflearn.

For example, a simple model like this: 

    embedding_layer = input_data(shape=[None,max_seq_len], name='input')
    net = embedding(
    	embedding_layer,
    	input_dim=vocab_size,
    	output_dim=embedding_dim,
    	weights_init = tf.convert_to_tensor(embedding_matrix, dtype='float32'),
    	trainable=False,
    	name="EmbeddingLayer")
    net = fully_connected(embedded_sequences, class_count, activation='softmax')
    net = regression(
    	net,
    	optimizer=optimizer,
    	loss='categorical_crossentropy')
    model = tflearn.DNN(net, tensorboard_verbose=0)

When I use `model.fit`:

    model.fit(
        X, one_hot_y,
        n_epoch=epochs,
        batch_size = batch_size,
        show_metric=True)

it prints the results per step: 

	---------------------------------
	Run id: E2VXW6
	Log directory: /tmp/tflearn_logs/
	INFO:tensorflow:Summary name Accuracy/ (raw) is illegal; using Accuracy/__raw_ instead.
	INFO:tensorflow:Summary name Accuracy/ (raw) is illegal; using Accuracy/__raw_ instead.
	---------------------------------
	Training samples: 18821
	Validation samples: 0
	--
	Training Step: 1  | time: 0.035s
	| Adam | epoch: 001 | loss: 0.00000 - acc: 0.0000 -- iter: 00032/18821
	Training Step: 2  | total loss: 3.22912 | time: 0.039s
	| Adam | epoch: 001 | loss: 3.22912 - acc: 0.0844 -- iter: 00064/18821
	Training Step: 3  | total loss: 4.40546 | time: 0.043s
	| Adam | epoch: 001 | loss: 4.40546 - acc: 0.0665 -- iter: 00096/18821
	Training Step: 4  | total loss: 4.53110 | time: 0.047s
	| Adam | epoch: 001 | loss: 4.53110 - acc: 0.0401 -- iter: 00128/18821
	Training Step: 5  | total loss: 4.18489 | time: 0.051s
        ....

I have tried to change parameters like `snapshot_epoch ` or `snapshot_step` but I don't think these have any control over it. I even tried to code a callback class, but still prints per step. 

How can I print **per epoch** instead?

**Comment by jrzaurin** [comment]
I see this is address in [here](https://github.com/tflearn/tflearn/issues/852) and also [here](https://github.com/tflearn/tflearn/issues/356) and the solution it seems to be changing the source of `tflearn`.
